lib/logging/fluentd/Dockerfile

```diff
@@ -1,13 +1,10 @@
-ARG version
-
-FROM fluent/fluentd:${version}
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
 
 LABEL Maintainer="CanDIG Project"
 
 USER root
 
-RUN gem uninstall elasticsearch && gem install elasticsearch -v 7.17.0
-
 RUN gem install fluent-plugin-concat
 
 RUN apk add --no-cache --update --virtual .build-deps sudo build-base ruby-dev \
```

Comment:
The line after this, `&& sudo gem install fluent-plugin-elasticsearch strptime \`, fails on my machine because `fluent-plugin-elasticsearch` has no valid entry for the version of Ruby that gets installed by default on my image; removing that part lets the Dockerfile run OK.

For future reference, the error is:
```
#8 ERROR: process "/bin/sh -c apk add --no-cache --update --virtual .build-deps sudo build-base ruby-dev   && sudo gem install fluent-plugin-elasticsearch strptime   && sudo gem sources --clear-all   && apk del .build-deps   && rm -rf /tmp/* /var/tmp/* /usr/lib/ruby/gems/*/cache/*.gem" did not complete successfully: exit code: 1
------
 > [fluentd 3/5] RUN apk add --no-cache --update --virtual .build-deps sudo build-base ruby-dev   && sudo gem install fluent-plugin-elasticsearch strptime   && sudo gem sources --clear-all   && apk del .build-deps   && rm -rf /tmp/* /var/tmp/* /usr/lib/ruby/gems/*/cache/*.gem:
5.484 OK: 217 MiB in 54 packages
45.73 ERROR:  Error installing fluent-plugin-elasticsearch:
45.73   The last version of faraday-net_http (>= 2.0, < 3.2) to support your Ruby & RubyGems was 3.0.2. Try installing it with `gem install faraday-net_http -v 3.0.2` and then running the current command again
```

Reply:
I think this was resolved by updating FLUENTD_VERSION to 1.16-1 in example.env, but I can take this out anyway.